snippets/model.cson

```diff
@@ -0,0 +1,52 @@
+'.source.coffee':
+
+
+  'DS.Model.extend(...':
+    'prefix': 'model'
+    'body':
+      """
+        _export = DS.Model.extend
+          ${1}
+      """
+
+
+
+
+
+  'Verbose DS.attr(...':
+    'prefix': 'model_attr-v'
+    'body': "${1}: DS.attr(  'string' ,  defaultValue:${2:null}  )     # or 'number' , 'boolean' , 'date'"
+
+
+  'DS.attr(...':
+    'prefix': 'model_attr'
+    'body': "${1}: DS.attr()"
+
+
+
+
+
+
+  'Verbose DS.hasMany(...':
+    'prefix': 'model_hasMany-v'
```

Comment:
it give the user an option to choose the simple or complex version.
"-v" is usually used in command line (terminal) as a flag, so I use it here.